core/org.openjdk.jmc.flightrecorder.writer/src/main/java/org/openjdk/jmc/flightrecorder/writer/AnnotationElementBuilderImpl.java

```diff
@@ -0,0 +1,32 @@
+package org.openjdk.jmc.flightrecorder.writer;
```

Comment:
Copyright header (Oracle + DD).

Reply:
😿  I really need to set up some templates. Always forgetting about this :(